client/index.html.tpl

```diff
@@ -18,11 +18,29 @@
 
 	<title>The Lounge</title>
 
-	<link rel="shortcut icon" href="img/favicon.png" data-other="img/favicon-notification.png" data-toggled="false" id="favicon">
-	<link rel="apple-touch-icon" sizes="120x120" href="img/apple-touch-icon-120x120.png">
-	<link rel="mask-icon" href="img/logo.svg" color="<%- themeColor %>">
+	<!-- Browser tab icon -->
+	<link rel="shortcut icon" href="img/favicon-normal.ico" data-other="img/favicon-alerted.ico" data-toggled="false" id="favicon" type="image/x-icon">
```

Comment:
Add `sizes="16x16 32x32 64x64"`.
Remove `shortcut ` as it's legacy just for IE.

Reply:
Done!